doc/man/s3fs.1

```diff
@@ -219,6 +219,9 @@ Because s3fs always checks MD5 when uploading large object, this option does not
 \fB\-o\fR iam_role ( default is no IAM role )
 This option requires the IAM role name or "auto". If you specify "auto", s3fs will automatically use the IAM role names that are set to an instance. If you specify this option without any argument, it is the same as that you have specified the "auto".
 .TP
+\fB\-o\fR ibm_auth ( default is not using IBM authentication )
+This option instructs s3fs to use IBM IAM authentication. In this mode, the AWSAccessKey and AWSSecretKey will be used as IBM's Service-Instance-ID and APIKey, respectively.
```

Comment:
Should this describe the service, e.g., IBM Bluemix COS, IBM Cloud, or whatever the preferred name is?  Repeated below.

Reply:
I think "IBM IAM authentication" is the correct description.
I will fix this instance (did not see any other).

Reply:
I agree that this is correct, but I'm more concerned that a user will see the flag and not associate it with IBM's cloud offering.  Adding some aside like (used with IBM Cloud) might help.